index.html

```diff
@@ -0,0 +1,193 @@
+              <hr>
+              <ul class="fa-ul">
+                <li>              
+                  <h2 class="card-price text-center"><b>$600</b></h2>
+                </li>
+                <li class="text-center">Lorem Ipsum</li>
+                <li class="text-center">Lorem Ipsum</li>
+                <li class="text-center"></span>Lorem Dolor</li>
+                <li class="text-center">Lorem Ipsum</li>
+              
+              </ul>
+              <div class="text-center">
+                <a href="#" class="btn btn-block btn-warning text-uppercase">SELECT</a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- Plus Tier -->
+        <div class="col-lg-4">
+          <div class="card mb-5 mb-lg-0">
+            <div class="card-body">
+              <h5 class="card-title text-muted text-uppercase text-center bg-light">Bass Trombone</h5>
+              <hr>
+              <ul class="fa-ul">
+                <li>              
+                  <h2 class="card-price text-center"><b>$900</b></h2>
+                </li>
+                <li class="text-center">Lorem Ipsum</li>
+                <li class="text-center">Lorem Ipsum</li>
+                <li class="text-center"></span>Lorem Dolor</li>
+                <li class="text-center">Lorem Ipsum</li>
+
+               
+              
+              </ul>
+              <div class="text-center">
+                <a href="#" class="btn btn-block btn-warning text-uppercase">SELECT</a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- Pro Tier -->
+        <div class="col-lg-4">
+          <div class="card">
+            <div class="card-body">
+              <h5 class="card-title text-muted text-uppercase text-center bg-light">Valve Trombone</h5>
+              <hr>
+              <ul class="fa-ul">
+                <li>              
+                  <h2 class="card-price text-center"><b>$1200</b></h2>
+                </li>
+                <li class="text-center">Plays similar to a Trumpet</li>
+                <li class="text-center">Great for Jazz Bands</li>
+                <li class="text-center"></span>Lorem ipsum dolor.</li>
+                <li class="text-center">Lorem Ipsum</li>
+              </ul>
+              <div class="text-center">
+                <a href="#" class="btn btn-block btn-warning text-uppercase">SELECT</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!--Pricing table ends-->
+    <footer class="pt-4 my-md-5 pt-md-5 border-top" style="padding-left: 30px; padding-right: 30px;">
+      <div class="row">
+        <div class="col-12" style="text-align: right;">
+          <h5><b>Privacy</b> &nbsp;&nbsp;&nbsp; <b>Terms</b> &nbsp;&nbsp;&nbsp;<b>Contract</b></h5>
+          Copyright 2016, Original Trombones
+        </div>
```

Comment:
It would be nice if you used the unordered list and proper styling to format this section of the footer for better HTML sematics. 👏🏾